packages/components/pie-cookie-banner/package.json

```diff
@@ -37,7 +37,7 @@
   "license": "Apache-2.0",
   "devDependencies": {
     "@custom-elements-manifest/analyzer": "0.9.0",
-    "@justeat/pie-design-tokens": "6.3.1",
+    "@justeat/pie-design-tokens": "6.5.0",
```

Comment:
We don't need to list the dependency here as it's indirectly imported from `@justeattakeaway/pie-css`. 

@xander-marjoram can you double-check that, please?